### ISSUE TITLE
CMake generates invalid project files for Visual Studio 12

### ISSUE THREAD

**Issue by danijar** [SFML/SFML#559]
**tl;dr solution:** Update or reinstall CMake to its latest version and use the latest version of SFML.

---

I used CMake to generate Visual Studio 12 project files for static linking. But those of the Audio, Network, Graphics and Window modules were invalid.

First, the projects can't be loaded at all because values in lines 79 and 111 respectively are messed up with `&quot` and additional quotation marks.

Second, the `<AdditionalOptions>` tag is used for linking against third party libraries whereas `<AdditionalDependencies>` would be correct. This is wrong for all configurations.

After I manually edited the four project files to fix both issues, static libraries could be generated without any errors. By the way, the Main and System projects were just fine.
